WordPress/Classes/Models/ReaderPost.m

```diff
@@ -65,6 +65,12 @@ - (BOOL)isCrossPost
     return self.crossPostMeta != nil;
 }
 
+- (BOOL)isAtomic
+{
+    // This is temporary until we start parsing a new value from the reader endpoint.
```

Comment:
Q: Do we wanna file an issue for follow up? Can we add a `// TODO` for tracking purposes?

Reply:
I updated this to be a "TO DO" and to include a link to the issue.